WebDriverAgentLib/Utilities/FBClassChainQueryParser.h

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import <XCTest/XCTest.h>
+
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface FBClassChainElement : NSObject
+
+/*! Element's position */
+@property (readonly) NSInteger position;
```

Comment:
you should make all properties `nonatomic` unless you really, really want them `atomic`.